src/main/webapp/app/account/user/settings/credential-revocation-settings/credential-revocation-settings.component.html

```
@@ -0,0 +1,55 @@
+<div class="d-flex flex-column">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)